moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js

```diff
@@ -32,200 +16,46 @@ import { getUserFullName } from "src/utils/userNames";
  * @constructor
  */
 const ProjectSummaryStatusUpdate = ({ projectId, data, refetch, classes }) => {
-  const userSessionData = getSessionDatabaseData();
-  const userId = userSessionData.user_id
-  const addedBy = getUserFullName(userSessionData);
-
-  const [updateProjectStatusUpdateInsert] = useMutation(
-    PROJECT_SUMMARY_STATUS_UPDATE_INSERT
-  );
-
-  const [updateProjectStatusUpdateUpdate] = useMutation(
-    PROJECT_SUMMARY_STATUS_UPDATE_UPDATE
-  );
-
-  /**
-   * Retrieve the last note made as a status update (which are ordered by date)
-   * @param {String} fieldName - The name of the field to retrieve in the moped_proj_note object
-   * @return {string|null}
-   */
-  const getStatusUpdate = fieldName => {
-    const lastItem =
-      (data?.moped_project[0]?.moped_proj_notes?.length ?? 0) - 1;
-    if (lastItem >= 0) {
-      // Get the data from the note
-      const note = data
-        ? data?.moped_project[0].moped_proj_notes[lastItem][fieldName] ?? ""
-        : null;
-      // Remove any HTML tags for status update string
-      return note ? parse(String(note)) : null;
```

Comment:
@johnclary I think this was the only place where the order being `asc` vs `desc` mattered.